src/openMVG/features/sift/SIFT_vlfeat_describer.hpp

```diff
@@ -0,0 +1,97 @@
+#pragma once
```

Comment:
As discussed before, all files are still protected by the license of the project. But we will run a script soon to update them and clarify the files that are part of the project.